app/controllers/project_database.py

```diff
@@ -369,8 +381,20 @@ def get(self, database_id):
 
         if errors:
             return dict(status='fail', message=errors), 500
+        
+        # Check the database status on host
+        database_service = DatabaseService()
+        database_connection = database_service.create_db_connection(user=database.user, password=database.password, db_name=database.name)
+
+        if not database_connection:
+            db_status='Failed'
```

Comment:
I think it is better if we make `db_status` a boolean since its either `true` or `false`

Reply:
I thought status of `Running` & `Failed` would make more sense to the frontend

Reply:
I have refactored these.